tests/Annotations/Command/QueueImportCommandTest.php

```diff
@@ -21,7 +21,7 @@
 /**
  * @covers \eLife\Annotations\Command\QueueImportCommand
  */
-class QueueImportCommandTest extends PHPUnit_Framework_TestCase
+final class QueueImportCommandTest extends PHPUnit_Framework_TestCase
```

Comment:
is `final` for tests something that happens in other projects too? We have to come up with a check for `proofreader` then